src/main/java/lv/ctco/javaschool/goal/control/GoalStore.java

```diff
@@ -83,4 +85,41 @@ public void addComment(Comment comment) {
         return tagSet;
     }
 
+    public List<Tag> getAllTagsForGoal(Goal goal) {
```

Comment:
Do you actually need this method? Can't `goal.getTags()` be used instead?

Reply:
removed the redundant method